composer.json

```diff
@@ -17,13 +17,12 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 || ^8.0",
         "nunomaduro/collision": "^5.0.0-BETA4",
         "pestphp/pest-plugin": "^0.3",
         "pestphp/pest-plugin-coverage": "^0.3",
         "pestphp/pest-plugin-init": "^0.3",
-        "phpunit/phpunit": "^9.2.6",
-        "sebastian/environment": "^5.1.2"
+        "phpunit/phpunit": "9.3.2"
```

Comment:
Just interested, is this intentionally a specific version rather than a range such as `^9.3`/`~9.3.2`? 🙂

Reply:
Yes, this is intentional, because we are using internal phpunit classes.